nixpkgs_review/review.py

```diff
@@ -143,6 +134,23 @@ def __init__(
         self.extra_nixpkgs_config = extra_nixpkgs_config
         self.num_parallel_evals = num_parallel_evals
 
+    def _process_aliases_for_systems(system: str) -> set[str]:
+        match system:
+            case "current":
+                return set([current_system()])
+            case "all":
+                return OFBORG_PLATFORMS
+            case "linux":
+                return OFBORG_PLATFORMS_LINUX
+            case "darwin" | "macos":
+                return OFBORG_PLATFORMS_DARWIN
+            case "x64" | "x86" | "x86_64" | "x86-64" | "x64_86" | "x64-86":
+                return OFBORG_PLATFORMS_X64
+            case "aarch" | "arm" | "aarch64" | "arm64":
+                return OFBORG_PLATFORMS_AARCH64
+            case _:
+                return set([system])
```

Comment:
Can we get those documented? Maybe a table in the README?

Reply:
Done!